src/primo/browzine-primo-adapter.js

```diff
@@ -529,6 +557,14 @@ browzine.primo = (function() {
       articlePDFDownloadLinkText = browzine.articleRetractionWatchText || "Retracted Article";
     }
 
+    if (showEocNoticeUI(articleEocNoticeUrl) && !showRetractionWatchUI(articleRetractionUrl)) {
```

Comment:
One thought: if this shouldn't happen when showRetractionWatchUI is truthy, there's at least another way we could change this to reduce a little code here:

```suggestion
    } else if (showEocNoticeUI(articleEocNoticeUrl)) {
```

By doing this, if we have `showRetractionWatchUI(articleRetractionUrl)` truthy, then the stuff on the above lines runs, but if that's falsy, then we give the EocNotice its attempt.